src/scripts/logBook/LogBook.ts

```diff
@@ -24,7 +26,7 @@ class LogBook implements Feature {
 
     toJSON(): { logs: Array<{ type: LogBookType; description: string; date: number }> } {
         return {
-            logs: this.logs.slice(0, 100),
+            logs: ko.unwrap(this.logs.slice(0, 100).map((v) => ko.unwrap(v))),
```

Comment:
I don't think this change is necessary. The individual logs themselves aren't wrapped in an observable, so we don't need the map, and `.slice` on a `KnockoutObservableArray<T>` will return an `Array<T>` making the outer unwrap unnecessary.

Reply:
Good catch. I've reverted that now :+1: